MediaAdminBundle/Resources/public/coffee/GalleryLoad.coffee

```diff
@@ -1,4 +1,5 @@
-GalleryLoad = (link, target) ->
+GalleryLoad = (link, target, mediaView) ->
```

Comment:
Are you sure of the order of the params ?
target can be undefined. Is it possible to call the GalleryLoad with no target defined and a mediaView defined? In that case, the mediaView will be received not as the mediaView, but as the target.